_data/our-team.yaml

```diff
@@ -100,6 +100,12 @@
   social: https://twitter.com/@lionsummerbell
 
 - type: current
+  name: Roxanne Joseph
+  image: /images/team/team_roxanne.jpg
+  pos: Data Journalist
```

Comment:
I just want to check - is this the title Rox prefers? Her contract calls her a campaign manager...